cmd/build-helm.go

```diff
@@ -84,5 +86,12 @@ func init() {
 		"Include memory limits when generating helm chart",
 	)
 
+	buildHelmCmd.PersistentFlags().StringP(
+		"version",
```

Comment:
Note that we already have a [`FISSILE_VERSION`](https://github.com/SUSE/scf/blob/10b4a5ccb4d09871923bcc60c6274b692ca1a589/bin/common/versions.sh#L12) variable, which will get picked up because of viper/cobra.  You might need a different name, or change that.

Reply:
x-ed, see my latest musing in main comment. Changing to `--tag-extra` now.